src/xenia/kernel/xam/apps/messenger_app.h

```diff
@@ -7,9 +7,6 @@
  ******************************************************************************
  */
 
-#ifndef XENIA_KERNEL_XAM_APPS_UNKNOWN_F7_APP_H_
```

Comment:
Keep the header guard, but fix the naming convention.

Reply:
got it

Reply:
you're missing `#endif` too